test/support/test_data.ex

```diff
@@ -0,0 +1,205 @@
+defmodule MultipassEx.TestData do
+  @moduledoc false
+  use ExUnitProperties
+
+  @dialyzer :no_return
+
+  def site_key do
+    gen all site_key <- string(:alphanumeric, min_length: 32) do
+      site_key
+    end
+  end
+
+  def api_key do
+    gen all shared_secret <- string(:alphanumeric, min_length: 32) do
+      shared_secret
+    end
+  end
+
+  def example_structure do
+    gen all map <-
+              fixed_map(%{
+                "idk" =>
+                  fixed_map(%{
+                    "needed" => string(:alphanumeric),
+                    "strings" => string(:alphanumeric),
+                    "for" => string(:alphanumeric),
+                    "this" => string(:alphanumeric),
+                    "to" => string(:alphanumeric),
+                    "test" => boolean(),
+                    "somewhat" => string(:alphanumeric)
+                  }),
+                "complex" => list_of(fixed_map(%{})),
+                "structure" => string(:alphanumeric),
+                "with" =>
+                  list_of(
+                    fixed_map(%{
+                      "a" => string(:alphanumeric),
+                      "round" => string(:alphanumeric),
+                      "trip" => string(:alphanumeric),
+                      "of" => boolean()
+                    })
+                  ),
+                "this" => string(:alphanumeric),
+                "." => boolean()
+              }) do
+      map
+    end
+  end
+
+  def pseudo_encrypted_string do
+    gen all string <-
+              string(Enum.concat([?a..?z, '+', '\n', '/']), min_length: 60) do
```

Comment:
i believe you could pre-compute the `Enum.concat` in a module variable to avoid calculating it with every test.